rules/default.js

```diff
@@ -1,110 +1,115 @@
 module.exports = {
-  "parser": "babel-eslint",
-  "env": {
-    "browser": true,
-    "node": true,
-    "es6": true
+  parser: 'babel-eslint',
+  env: {
+    browser: true,
+    node: true,
+    es6: true
   },
-  "plugins": [
-    "react",
-    "no-copy-paste-default-export",
-    "no-loops"
+  plugins: [
+    'react',
+    'no-copy-paste-default-export',
+    'no-loops'
   ],
-  "ecmaFeatures": {
-    "jsx": true,
-    "modules": true
+  ecmaFeatures: {
+    jsx: true,
+    modules: true
   },
-  "extends": "eslint:recommended",
-  "rules": {
+  extends: 'eslint:recommended',
+  settings: {
+    react: {
+      version: '0.14.0'
```

Comment:
nothing, deprecations warnings are updated on majors so 0.14.x have all the same warnings (in fact I don't think you can specify the minor in these settings)